From c380e0dae62d7df98f7c94c7f7ab59ee406dbfe3 Mon Sep 17 00:00:00 2001
From: Robby Cai <r63905@freescale.com>
Date: Tue, 29 Sep 2015 18:25:30 +0800
Subject: [PATCH 0631/5242] MLK-11658 ARM: dts: imx7d-12x12-lpddr3-arm2:
 enable v4l2 output

commit  5e859ea2f295b836da9a9637cb9b62090a2dd3f0 from
https://source.codeaurora.org/external/imx/linux-imx.git

enable v4l2 output feature on i.MX7D 12x12 LPDDR3 ARM2 board.

Signed-off-by: Robby Cai <r63905@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
index c8f4482..b8eae33 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
@@ -43,7 +43,7 @@
 
 	pxp_v4l2_out {
 		compatible = "fsl,imx7d-pxp-v4l2", "fsl,imx6sx-pxp-v4l2", "fsl,imx6sl-pxp-v4l2";
-		status = "disabled";
+		status = "okay";
 	};
 
 	regulators {
-- 
1.7.9.5

